src/ashell/jerry-code.c

```diff
@@ -74,6 +120,100 @@ static void javascript_print_value(const jerry_value_t value)
     jerry_port_console("\n");
 }
 
+static int read_file(const char *file_name, char **file_buf)
```

Comment:
I think it would be safer for maintenance to return file_buf and take int *size as an argument instead. That makes it more clear that you're doing a malloc inside, whereas the int is not something you would leak. Also then you can optionally set the size only if they give you a real pointer.

`read_file_to_new_buffer` might make it really clear. :)